k8s/createLKE_install_PMM_HA.sh

```
@@ -0,0 +1,211 @@
+    fi
+done
+
+echo "Cluster ID: $CLUSTER_ID"
+
+#############################################
+# Wait Until Ready
+#############################################
+
+echo "Waiting for cluster to become READY..."
+
+[ $(linode-cli lke pools-list "$CLUSTER_ID" --json | jq '.[] | .nodes[] | select(.status == "ready")' | jq -s 'length') -eq $NODE_COUNT ] && echo "Cluster is READY"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)